data/pages-content/cfp.md

```diff
@@ -0,0 +1,72 @@
+---
+title: CFP
+subtitle: EuroPython Call for Proposals
+---
+# Call for Proposals
+
+🐍 Pythonistas, mark the dates:
+
+<div style={{textAlign: "center"}}>
+**EuroPython 2023 Call for Proposals (CFP) will be open** between
+
+ **Monday, 6 March 2023** and
+
+ **Sunday, 19 March 2023** [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+</div>
+
+
+
+We invite proposals from the whole community on a wide range of topics. We welcome speakers at all python levels with any experience with public speaking.
+
+## The Conference Format ##
+
+The conference will feature around 16 Tutorials/Workshops and 120 talk sessions:
+
+* **Tutorials/Workshops (17-18 July)**: hands-on **180-mins sessions**
+
+* **Talks (19-21 July)** : **30, 45, or 60-minute** presentations of Python general topics & PyData related topics
+
+All presentation content can be at **beginner, intermediate or advanced** level.
+
+[comment]: # (Other than talks and tutorials, you are also welcome to submit **posters**: printed in large formats, it is a graphical way to showcase a project or a technology. Posters are exhibited at the conference, read at any time by participants, and can be discussed face to face with their authors during the poster session.)
```

Comment:
I think It's safe to also advertise posters. I'd maybe tweak the wording to add research as well. So something to the tune of "it is a graphical way to showcase your search, projects or a technology."